tiatoolbox/annotation/storage.py

```diff
@@ -2143,22 +2218,26 @@ def _initialize_query_string_parameters(
             query_parameters["where"] = where
         # Predicate is a string
         if isinstance(where, str):
-            sql_predicate = eval(where, SQL_GLOBALS, {})  # skipcq: PYL-W0123
+            sql_predicate = eval(  # skipcq: PYL-W0123,  # noqa: PGH001
+                where,
+                SQL_GLOBALS,
+                {},
+            )
             query_string += f" AND {sql_predicate}"
 
         return query_string, query_parameters
 
     def _query(
         self,
         columns: str,
-        geometry: Optional[Geometry] = None,
-        callable_columns: Optional[str] = None,
+        geometry: Geometry | None = None,
+        callable_columns: str | None = None,
         geometry_predicate="intersects",
-        where: Optional[Predicate] = None,
+        where: Predicate | None = None,
         unique: bool = False,
         no_constraints_ok: bool = False,
         index_warning: bool = False,
-        min_area: Optional[float] = None,
+        min_area: float | None | None = None,
```

Comment:
None is repeated

Reply:
Addressed in this commit https://github.com/TissueImageAnalytics/tiatoolbox/pull/625/commits/991322d859c0c22e7c7a1d713362944aa4317198